plugins/webp-uploads/hooks.php

```diff
@@ -397,7 +424,12 @@ function webp_uploads_remove_sources_files( int $attachment_id ): void {
 				continue;
 			}
 
-			if ( ! is_array( $properties ) || empty( $properties['file'] ) ) {
+			if (
+				! is_array( $properties ) ||
+				! isset( $properties['file'] ) ||
```

Comment:
The 2nd check satisfies the 1st:
```suggestion
				! isset( $properties['file'] ) ||
```